### ISSUE TITLE
Sklearn中Scale, Standardize, Normalize的区别 | BrambleXu

### ISSUE THREAD

**Issue by LearnXu** [LearnXu/blog-comment#27]
https://bramblexu.org/2020/04/28/20200428-sklearn-zhong-scale-standardize-normalize-de-qu-bie/ 

 Sklearn中MinMaxScaler, RobustScaler, StandardScaler, and Normalizer的区别

**Comment by pleomax0730** [comment]
這裡最後一點是否有誤? standardscaler應該是無法抵銷outlier的效果的，https://scikit-learn.org/stable/auto_examples/preprocessing/plot_all_scaling.html#standardscaler
官方最後一段寫道:
StandardScaler therefore cannot guarantee balanced feature scales in the presence of outliers.

如果要減少outlier的影響，應該使用RobustScaler比較好一些。

**Comment by BrambleXu** [comment]
> @pleomax0730
> 這裡最後一點是否有誤? standardscaler應該是無法抵銷outlier的效果的，https://scikit-learn.org/stable/auto_examples/preprocessing/plot_all_scaling.html#standardscaler
> 官方最後一段寫道:
> StandardScaler therefore cannot guarantee balanced feature scales in the presence of outliers.
> 
> 如果要減少outlier的影響，應該使用RobustScaler比較好一些。

谢谢你提供的文档，根据分布图来看来，确实RobustScaler更好一些。看来[原文](https://towardsdatascience.com/scale-standardize-or-normalize-with-scikit-learn-6ccc7d176a02)中关于StandardScaler的笔记作者也理解错了。
之后我会更新文章里相应的内容。
